### PR TITLE
Feature/configure level for student

### PR DESCRIPTION
**Pull request by PavloJedi**

### CODE REVIEW

File: front-end/src/components/Students/Table/Table.module.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
+@import 'styles/colors.scss';
+@import 'styles/font.scss';
+
+table {
```
Comment (andriydzhrvts) [inline front-end/src/components/Students/Table/Table.module.scss:4]: Are you sure that these styles only for your table not for all tables in app?

File: front-end/src/components/Students/Table/index.jsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                      id='basic-menu'
+                      anchorEl={anchorEl}
+                      open={open}
+                      onClose={handleClose}
+                      MenuListProps={{
+                        'aria-labelledby': 'basic-button',
+                      }}
+                    >
+                      <MenuItem
+                        onClick={async () => {
```
Comment (andriydzhrvts) [inline front-end/src/components/Students/Table/index.jsx]: Create method for this onClick

File: front-end/src/components/Students/Table/index.jsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+import ChangeLevel from 'components/ChangeLevel/ChangeLevel';
+
+// Images
+import teacher from 'assets/images/teacher1.jpg';
+
+// Styles
+import styles from './Table.module.scss';
+import { Menu, MenuItem } from '@mui/material';
+
+const Table = () => {
```
Comment (andriydzhrvts) [inline front-end/src/components/Students/Table/index.jsx:25]: Why did you create a new table? As I know we already have a table for subscriptions.

Comment (PavloJedi) [reply front-end/src/components/Students/Table/index.jsx:25]: That table was created by Nadia, i just added my logic here

File: front-end/src/components/Students/Table/Table.module.scss (hunk 1 above)
Comment (PavloJedi) [reply front-end/src/components/Students/Table/Table.module.scss:4]: this style will be changed, after Sasha's style update on this table

File: front-end/src/components/Students/Table/index.jsx (hunk 2 above)
Comment (PavloJedi) [reply front-end/src/components/Students/Table/index.jsx]: made method

File: front-end/src/components/Students/Table/index.jsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    try {
+      await deleteSubscription(subscriptionId);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    fetchSubscriptions(currentUser.roleId);
+  }, [currentUser]);
```
Comment (DeLettenhove) [inline front-end/src/components/Students/Table/index.jsx]: Don't use objects in dependecy array

**Review by andriydzhrvts — APPROVED** [review]

**Pull request merged**